CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Flash messages on a successful login and on errors catched by the global vue error handler ([#7], [#16],[#44],[#43])
 - Added Footer with Impress and Privacy policy ([#9], [#17])
 - Added authenticator type to users model ([#19], [#21])
+- Added Initial Admin Pages (users, rooms, records, roles, etc. management)([#10], [#23])
```

Comment:
missing link to `#10`

Reply:
https://github.com/THM-Health/PILOS/pull/23/commits/1a33a7aa89dddde61afd7c0994421ca8d1e20bfb added the link